crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -68,6 +68,7 @@ type CheckpointExecutionBuffer = FuturesOrdered<JoinHandle<VerifiedCheckpoint>>;
 
 /// The interval to log checkpoint progress, in # of checkpoints processed.
 const CHECKPOINT_PROGRESS_LOG_COUNT_INTERVAL: u64 = 5000;
+const SCHEDULING_EVENT_FUTURE_TIMEOUT_MS: u64 = 200;
```

Comment:
Does this interval need to be shorter than checkpoint creation rate?

Reply:
It needs to be longer than the checkpoint rate, as otherwise we'd sometimes log due to no new checkpoints. I commented about this somewhere in slack, and it was agreed that 1 second should be sufficient, although it won't fix the simtest issue that this addresses (since that simtests has 1 second long epochs). There is a better fix for that elsewhere

Reply:
Optionally you can make it small for simtest only, to fix the test until a proper fix is here.